### ISSUE TITLE
Have you tried to train a model for 44.1KHz?

### ISSUE THREAD

**Issue by lyapple2008** [Xiao-Ming/UNet-VocalSeparation-Chainer#6]
Since the music always be saved as the sample rate of 44.1KHz, have you tried to train a model for 44.1KHz?

**Comment by Xiao-Ming** [comment]
Not yet, but it shouldn't be too difficult. I'll try on 44.1 kHz / stereo when my current task is over:)

**Comment by lyapple2008** [comment]
Sure, I had implement deep u-net using keras. And I found that the smaller loss did not mean better separation performance. Do you think what is the reason? And I could not get better performance by giving more training data.

**Comment by Xiao-Ming** [comment]
I'm not sure, since I did not do any quantitative evaluation on the separated results. I did confirm that the trained model doesn't work well in some scenarios like hip-hops or noisy speeches, though. I think it's simply an over-fitting issue.
